pkg/swarm/swarm.go

```diff
@@ -83,3 +84,33 @@ func (a Address) MarshalJSON() ([]byte, error) {
 
 // ZeroAddress is the address that has no value.
 var ZeroAddress = NewAddress(nil)
+
+
+// Chunk defines a swarm chunk structure. It contains of the Address and the
+// chunk data.
+type Chunk struct {
```

Comment:
Maybe be explicit and call this an addressed chunk (don't want to review this now, but scanned over it and saw it). Good work so far! :)

Reply:
Single Owner chunk will be called as SOC to avoid confusion. Normal chunk is chunk to maintain backward compatibiity.